backend/src/packages/articles/article.repository.ts

```diff
@@ -35,12 +35,14 @@ class ArticleRepository implements IRepository {
       ArticleEntity.initializeWithAuthor({
         ...article,
         genre: article.genre.name,
-        prompt: {
-          character: article.prompt.character,
-          setting: article.prompt.setting,
-          situation: article.prompt.situation,
-          prop: article.prompt.prop,
-        },
+        prompt: article.prompt
+          ? {
+              character: article.prompt.character,
+              setting: article.prompt.setting,
+              situation: article.prompt.situation,
+              prop: article.prompt.prop,
+            }
+          : undefined,
```

Comment:
Pass `null` explicitly instead of undefined

Reply:
It's an optional parameter that's why undefined.  In order to pass null I will have to change type and rewrite all initialize methods in article entity